Add missing linux/param.h header include

Fixes the following build failure under musl:

mem_if.c: In function ‘MapPhysicalMemory’:
mem_if.c:337:36: error: ‘EXEC_PAGESIZE’ undeclared (first use in this function)
  ulDiff = (ULONG)(tdStartAddress % EXEC_PAGESIZE);
                                    ^

Signed-off-by: Baruch Siach <baruch@tkos.co.il>

diff -Nuar ipmiutil-2.9.5-orig/util/mem_if.c ipmiutil-2.9.5/util/mem_if.c
--- ipmiutil-2.9.5-orig/util/mem_if.c	2014-11-04 19:46:11.000000000 +0200
+++ ipmiutil-2.9.5/util/mem_if.c	2015-09-29 08:08:31.800459775 +0300
@@ -65,6 +65,9 @@
 #include <sys/param.h>
 #include <sys/mman.h>
 #include <sys/ioctl.h>
+#ifdef __linux__
+#include <linux/param.h>
+#endif
 #endif
 #if defined(SOLARIS) || defined(BSD)
 #define EXEC_PAGESIZE     4096
